docs/components/input-field/demo/base-demo.md

```diff
@@ -16,18 +21,13 @@ import { tracked } from '@glimmer/tracking';
 
 export default class extends Component {
   @tracked value;
-  @tracked errorMessage;
+  @tracked count = 0;
 
   @action
-  updateValue(value, e) {
-    this.value = value;
+  handleChange(value, event) {
+    console.log({ value, event });
 
-    if (value !== 'input') {
-      this.errorMessage = 'Input must match "input"';
-      return;
-    }
-
-    this.errorMessage = null;
```

Comment:
Any reason to remove this block that would apply the error state? Otherwise it's a bit tricky to test the error state + character count together.